openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -751,7 +751,31 @@ body .profiler-graph .recharts-active-dot circle {
 }
 #center {
   grid-area: centerarea;
+  overflow-y: auto;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
```

Comment:
Do we really need this to be added as a comment?

Reply:
Yeah, this is related to the scrollbar so will remove it after discussion.